### PR TITLE
ActivePageLogic

### PR DESCRIPTION
**Pull request by cannarelladev**
# Description

Download and show real data for the Active Page using Crownlabs APIs. improve and fix Types and components Interfaces

# TODO

- [x] User view
- [x] Manager view
- [x] destroy function

### CODE REVIEW

File: frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  useEffect(() => {
+    if (!loadingInstances) {
+      subscribeToMoreInstances({
+        document: updatedOwnedInstances,
+        variables: { tenantNamespace },
+        updateQuery: (prev, { subscriptionData }) => {
+          const {
+            data,
+          } = subscriptionData as UpdatedOwnedInstancesSubscriptionResult;
+
```
Comment (QcFe) [inline frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx:54]: Isn't this logic quite similar to the one which @cheina97 made? Can't we find a way to recycle it? Or at least parts of the code in this hook?

File: frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+          data,
+        } = subscriptionData as UpdatedInstancesLabelSelectorSubscriptionResult;
+
+        if (!data?.updateInstanceLabelSelector?.instance) return prev;
+
+        const { instance, updateType } = data?.updateInstanceLabelSelector;
+
+        if (prev.instanceList?.instances) {
+          let instances = prev.instanceList.instances;
+          if (updateType === 'DELETED') {
```
Comment (QcFe) [inline frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx:135]: There should be an enum for the updateType

**Review by QcFe — COMMENTED** [review]
Some comments inline

File: frontend/src/components/activePage/TableTemplate/TableTemplateRow.tsx [hunk 3]
```diff
@@ -40,11 +40,23 @@ const TableTemplateRow: FC<ITableTemplateRowProps> = ({ ...props }) => {
...
+                <div className="text-center">
+                  Your files won't be deleted in case of an internal misservice
+                  of CrownLabs.
+                </div>
+              </>
+            }
+          >
+            <div className="success-color-fg flex items-center">
+              <SvgInfinite width="22px" />
+            </div>
```
[the remark this answers is not in the document]
Comment (QcFe) [reply frontend/src/components/activePage/TableTemplate/TableTemplateRow.tsx:59]: @cannarelladev we still have this one open :)

File: frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx (hunk 1 above)
Comment (cannarelladev) [reply frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx:54]: we could try to do it in the other PR where we clean and organize the code removing also duplicates

**Review by QcFe — APPROVED** [review]

**Pull request merged**